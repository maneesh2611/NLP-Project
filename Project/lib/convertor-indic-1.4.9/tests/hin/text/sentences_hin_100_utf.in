हरिद्वार में माँ गंगा एक सिरे से दूसरे सिरे तक बहती हैं, लेकिन जो पुण्य हर की पौड़ी में स्थित ‘ब्रह्मकुंड’ में स्नान से मिलता है, वह कहीं नहीं मिलता।
माना जाता है कि अमृतमंथन के बाद अमृत की कुछ बूँदें यहाँ गिरी थीं, इसलिए इसे ब्रह्मकुंड कहा जाता है।
यहाँ  पर ट्रस्ट के द्वारा यात्रियों की सुविधा के लिए उड़नखटोला (रोप वे) बनाया गया है, जिससे दो जगहों पर यात्री कुछ समय में ही माता के दर्शन कर लौट सकते हैं।
इसलिए इस झील को शेषनाग झील कहा जाता है।
धरती पर कहीं स्वर्ग है, तो वह कश्मीर की खूबसूरत वादियाँ ही हैं।
हम यहाँ पर कुछ लोगों को रोमान्चकारी नौकायान के लिए प्रशिक्षित करेंगे, जिससे वह अपनी जीविका का साधन भी बना सकेगें।
मगर नदी का वास्तविक शान्त स्वरूप, उसकी वास्तविक सुन्दरता है।
उसी का प्रमाण है कि अब भी विदेशी पर्यटक भारत की कला को न केवल देखने आते हैं, बल्कि उसके आगे नतमस्तक भी हो जाते हैं।
अगर आप विदेश से चेन्नई के लिए उड़ान भर रहे हैं तो हाइवे पर अंतरराष्ट्रीय हवाई अड्डा आता है।
उत्तरायण सूर्य के समय 21 जून को यहाँ सूर्य नहीं छिपता है तथा चौबीसों घंटे दिन रहता है।
यह शहर आर्कटिक सर्कल के उत्तर में एक टापू पर बसा है तथा चारों ओर पहाड़ों, फ्यूर्डो तथा द्वीपों से घिरा है।
यदि आप स्वच्छ हवा, निर्मल पानी और खूबसूरत वादियों के मुरीद हैं तो स्कॉटलैंड आपकी मंजिल है।
यदि आप भीड़-भाड़ से दूर एकांत में प्रकृति की अनुपम छटा को निहारना चाहते हैं तो स्कॉटलैंड के पास आपको देने के लिए काफी कुछ है।
यहाँ गोल्फ खिलाया ही नहीं जाता बल्कि सिखाया भी जाता है।
पहली मंजिल पर शानदार होटल है, दूसरी पर  कैफे और तीसरी मंजिल को सुंदर नजारों को निहारने के लिए खाली छोड़ा गया है।
मीरा यहाँ अपने कृष्ण की पूजा करती थी और यहीं पर उन्होंने अमृत मानकर विष का प्याला पी लिया था।
स्तूप क्रमांक दो पहाड़ी के शिखर पर स्थित है और यह पत्थर की सुंदर चारदीवारी से घिरा हुआ है।
यहां लगभग 75 मंदिर हैं और इनके बारे में अलग-अलग किंवदंतियां हैं।
वर्षभर पर्वतीय शिखर वर्षा से आच्छादित रहते हैं जबकि मैदानी भाग  रेगिस्तान है।
किंवदंतियों के अनुसार शिव शंभु अमर हैं, किंतु सती के साथ जीवन-मरण का बंधन चलता रहता था।
पर्यटन मन्त्रालय का मानना है कि इस तरह के रोमान्चकारी खेलों को बढ़ावा देना बहुत आवश्यक है।
ऐसा लगे कि हम भगवान श्री हरि विष्णु के नगर में पहुँच गए हैं।
यहाँ पर ही विश्व प्रसिद्ध राम झूला एवं लक्ष्मण झूला नामक पुल हैं, जो गंगा नदी पर बने हैं।
गंगा के निर्मल जल की कल-कल ध्वनि से मुग्ध कर देने वाला संगीत पैदा होता है, जो यहाँ आने वाले हर व्यक्ति को अपनी ओर आकर्षित करता है।
जिस स्थान पर नंदी बैल को छोड़ा गया था, उस स्थान का नाम बैलगाँव पड़ा।
इस यात्रा में हजारों साधु तथा श्रद्धालु भी शामिल होते हैं जो भगवान शिव के लिंग के दर्शन करने की इच्छा लिए होते हैं।
मंदिर की चारदीवारी पर लकड़ी के काम की नक्काशी की गई है जो सदियों पुरानी बताई जाती है।
जब वे मदुरै की सीमा में पहुँचे तो उन्हें सूचना मिली कि विवाह संपन्न हो गया।
जब यह प्रख्यात वनस्पति वैज्ञानिक पैरोटेट की देख-रेख में आया तो यहाँ कलकत्ता, मद्रास, सीलोन तथा रूमानिया से पौधे लाए गए।
यह छोटा सा  सुंदर  गाँव  है,  जहाँ  से  कई ग्लेशियरों की यात्रा की जा सकती है।
जो लोग आगरा शहर में निर्मित स्थापत्य की बेजोड़ कलाओं को देखने आते हैं, वे एक बार फतेहपुर सीकरी का रुख जरूर करते हैं।
जो भी भक्त इन मंदिरों में दर्शन हेतु आते हैं, वे इनसे पूजा जरूर करवाते हैं।
जब पूरा राजस्थान गर्मी में तपता है और सारे जलस्रोत सूख जाते हैं, तब भी इस गौमुख से निकलने वाली धारा लगातार शिवजी का अभिषेक करती रहती है।
यह छोटा सा  सुंदर  गाँव  है,  जहाँ  से  कई ग्लेशियरों की यात्रा की जा सकती है।
यहाँ के ब्राइगन घाट पर लकड़ी के अनेक सुंदर मकान हैं, जो मध्यकालीन युग की याद दिलाते हैं।
यह मंदिर उसी स्थान पर बनाया गया है, जहाँ से यह मूर्ति निकाली गई थी।
मंदिर के चार बड़े-बड़े द्वार हैं, जो सभी दिशाओं की ओर बनाए गए हैं।
ठीक सामने मौजूद पैगोडा के ऊपर बँधी घंटियाँ सुरम्य और शांत वातावरण में जब बजती हैं तो लगता है कि ये सभी दिशाओं में अहिंसा और प्रेम का संदेश दे रही हों।
दक्षिण का दर्प-शिल्प सौंदर्ययुक्त भव्य मीनाक्षी मंदिर देखने के बाद लगा अब तक हम जिन्हें भव्य कहते आए थे, वे  इसके पासंग भी नहीं हैं।
प्रत्येक गोपुरम नौ मंजिला है, जो कि एक बड़े आयताकार में स्थित है।
अगले दिन जुलूस वैगा नदी तट पर जाता है, जहाँ अलगार मंदिर से अलगारजी की प्रतिमा नदी के दूसरे तट पर लाकर वापस ले जाई जाती है।
मूर्ति की पृष्ठभूमि में तोरण है, जिस पर दस अवतारों की लीला चित्रित है।
चम्बा में भूरीसिंह नाम का एक संग्रहालय है, जहां चम्बा घाटी की हर कला सुशोभित है।
इस तरह के नौकायान में कभी-कभी नई-नई जगहों का भी पता चल जाता है, जो अपने आप में बिल्कुल अलग है।
'हरिद्वार' का अर्थ है एक ऐसा स्थान, जहाँ पहुँचते ही एक अलग-सी अनुभूति हो।
नगर में चारों ओर भगवान के भजन गूँजते रहते हैं।
साल भर यहाँ श्रद्धालु आते रहते हैं।
रक्षा बंधन के नजदीक आते-आते यहाँ श्रद्धालुओं की भीड़ काफी बढ़ जाती है।
अमरनाथ की गुफा में हर साल एक प्राकृतिक हिमलिंग बनता है।
सौभाग्य से भीतर के मंदिर बच गए थे।
पांडिचेरी में बॉटनिकल गार्डन 1826 में बना।
त्रयोदशी के दिन पुंछ कस्बे के दशनामी अखाड़े से इस धर्मस्थल के लिए छड़ी मुबारक की यात्रा आरंभ होती है।
हिमालय से निकली गंगा नदी यहाँ से मैदानों की तरफ प्रवाहित होती है।
स 1996 में प्राकृतिक आपदा की वजह से गुफा के सामने हजारों लोगों की मौत हो गई।
जहाँ तत्कालीन बौद्ध भिक्षु निवास करते थे।
इसके  बाद शंभु ने माँ को अमरता की कथा सुनाई।
यह पावन सी सुबह कितना सुकून देती है।
श्री बाबा अमरनाथ श्राइन बोर्ड के द्वारा उन्हें चढ़ावे में से  कुछ  राशि  दी जाती है।
गुफा को 1950 में बूटा मलिक नामक मुस्लिम चरवाहे ने खोजा था।
वहाँ पर उन्होंने भगवान श्रीगणेश को विराजित किया।
सेना इसका संचालन करती है।
हर स्मारक एक कहानी बयान करता है।
मेरीटाइम म्यूजियम में आप अतीत के बड़े-बड़े जहाज देख सकते हैं।
राणा सांगा ने यहाँ रहकर अपनी फौज को मजबूत किया।
यूनेस्को ने इसे वैश्विक धरोहर घोषित किया है।
उसने भव्य इमारतों से मदुरै को सँवारा।
इसकी पुष्टि इससे भी होती है कि मुस्लिम बहुल क्षेत्र में स्थित इस मंदिर की रखवाली मुस्लिम ही करते हैं।
उसी के स्मारक के रूप में यहाँ सम्राट अशोक महान ने एक स्तूप का निर्माण भी किया।
सुंदरता, रहस्य, धर्म एवं विचित्र शांति का यह क्षेत्र पर्यटकों को मोह लेता है।
हरिद्वार के वातावरण में अनूठी पवित्रता और धार्मिकता नजर आती है।
इसके बाद से लोगों की बढ़ती भीड़ और वादी में बढ़ते आतंकवाद के कारण सरकार ने यहाँ सुरक्षा मुहैया करवाई।
इन तूफानों से प्रेरणा लेकर ओरलैंडों वासियों ने एक अनूठे भवन का निर्माण करवाया है और उसे नाम दिया  है - वन्डर वर्क्स।
यहाँ की सरकार हीरों के व्यवसाय को प्रमोट करने के लिए यहाँ फ्री विजिट करवाती है।
सलीम सिंह की हवेली को जैसलमेर के प्रधानमंत्री सलीम सिंह ने बनवाया था।
यह इतिहास तो बच्चे-बच्चे की जुबान पर है कि मुगल बादशाह शाहजहाँ ने अपनी दूसरी पत्नी  मुमताज महल की याद में ताजमहल का निर्माण करवाया था।
इसका निर्माण 1565 ई. में अकबर ने करवाया था।
बाबर ने इस बाग को 1528 ई. में बनवाया था।
इस  स्तंभ का निर्माण महाराणा कुंभा ने चौदहवीं शताब्दी में करवाया था।
यह महल राजा बीर सिंह जूदेव ने 17 वीं शताब्दी में जहाँगीर के ओरछा आने के पहले बनवाया था।
चतुर्भुज के आकार का यह महल 17 वीं शताब्दी में मधुकर शाह ने बनवाया था।
महाराजा ने ही यहाँ पदम तालाब, राजा बाँध और मिलक तालाब बनवाए।
बाद में बाहर वाले बड़े गोपुर फिर बनवाए गए।
यह मंदिर यहाँ के चोल राजाओं ने बनवाया था।
133 फुट ऊंची इस प्रतिमा को लगभग पाँच हजार शिल्प कर्मियों ने बेहद मेहनत से बनवाया था।
इसके साथ ही सेन नदी के किनारों पर चहलकदमी करते हुए फ्रेंच जीवन से रूबरू होना बेहद सुंदर अनुभव है।
बड़े और बच्चों दोनों  को ध्यान रखकर वॉटर स्लाइड्स बनाई गई हैं।
मीनाक्षी मंदिर को केंद्र में रखकर नगर की रचना कमल के फूल की आकृति में की गई है।
साथ ही इस तरह के रोमान्चकारी खेलों की विशेष व्यवस्था द्वारा पर्यटन के साथ-साथ स्थानीय लोगों के लिये रोजगार की भी अनेक  सम्भावनाएँ उभर कर आएगीं।
एक स्थान पर आकर भगवान भोलेनाथ ने नंदी बैल से उतरकर उसे वहीं छोड़ दिया और माता के साथ आगे की ओर निकल पड़े।
यह लेखा-जोखा हजारों वर्षों से वर्तमान पंडितों के पूर्वजों के द्वारा सँभालकर रखा गया है।
यहाँ से पहाड़ों के बीच से बहती हुई गंगा नदी का दर्शन बड़ा मनोरम प्रतीत होता है।
नौका में बैठकर या किनारे पर खड़े होकर मछलियों का दाना खिलाना भी अविस्मरणीय अनुभव रहेगा।
मन शांत, ताजा और प्रफुल्लित हो जाता है।
इसे ही बाद में पहलगाम कहा जाने लगा।
पहाड़ों से घिरा होने के कारण यह नगर अत्यंत ही सुंदर लगता है।
हरिद्वार को भारत की धार्मिक राजधानी माना जाता है।
इस स्थान का नाम चाँदवाड़ी पड़ा।
इस मूर्ति के चार मुख व चार हाथ हैं।
कहा जाता है कि रानी के प्रतिदिन नर्मदा दर्शन के पश्चात अन्न-जल ग्रहण करने की आदत के कारण बाज बहादुर ने यह ऊँचा महल बनवाया था ।
पिछले पन्द्रह वर्षों से इस क्षेत्र में वाहन चला रहे भानुप्रतापसिंह ने बताया कि पर्यटकों की संख्या निरंतर बढ़ रही है , लेकिन बाघ कभी कभार ही दिखता है ।
उन्होंने बताया कि पहले यहाँ बाघ दिखाने के लिए चार मार्ग थे , अब सात कर दिए गए हैं ।
खुले वाहनों में बिना किसी हथियार के पर्यटकों को जंगल में ले जाने के बावजूद दुर्घटना की आशंका से इनकार करते हुए एक अन्य वाहन चालक ने बताया कि यदि जंगल में गाड़ी में कोई खराबी हो जाए तो इसकी सूचना देने के लिए चालक को पैदल ही चौकी तक जाना होता है अथवा रास्ते से गुजर रहा कोई अन्य वाहन इसकी आगे सूचना देता है ।
कहा जाता है कि इन स्तूपों में बुद्ध और उनके शिष्यों के अवशेष रखे गए हैं ।
तिरुवनंतपुरम कुछ वैज्ञानिक और तकनीकी संस्थानों का भी केन्द्र है जिनमें विक्रम साराभाई स्पेस सेंटर , सेंटर फॉर अर्थ साइंस स्टडीज और एक ऐसा संग्रहालय है जोकि विज्ञान और प्रौद्योगिकी के सभी पहलुओं से साक्षात्कार कराता है ।
हिमालय में पश्चिम गढ़वाल के बर्फ से ढँके श्रंग बंदरपुच्छ जो कि जमीन से 20 , 731 फुट ऊँचा है , के उत्तर-पश्चिम में कालिंद पर्वत है ।
विशेषण आकर्षण यह है कि इस मंदिर के पास पवित्र गंगा नदी जो कि बंगाल में हुगली नदी के नाम से जानी जाती है , बहती है ।
प्रत्येक गोपुरम नौ मंजिला है , जो कि एक बड़े आयताकार में स्थित है ।
