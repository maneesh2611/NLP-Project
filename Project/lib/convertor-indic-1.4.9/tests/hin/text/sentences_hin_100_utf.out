harixvAra meM mAz gaMgA eka sire se xUsare sire waka bahawI hEM, lekina jo puNya hara kI pOdZI meM sWiwa ‘brahmakuMda’ meM snAna se milawA hE, vaha kahIM nahIM milawA.
mAnA jAwA hE ki amqwamaMWana ke bAxa amqwa kI kuCa bUzxeM yahAz girI WIM, isalie ise brahmakuMda kahA jAwA hE.
yahAz  para trasta ke xvArA yAwriyoM kI suviXA ke lie udZanaKatolA (ropa ve) banAyA gayA hE, jisase xo jagahoM para yAwrI kuCa samaya meM hI mAwA ke xarSana kara lOta sakawe hEM.
isalie isa JIla ko SeRanAga JIla kahA jAwA hE.
XarawI para kahIM svarga hE, wo vaha kaSmIra kI KUbasUrawa vAxiyAz hI hEM.
hama yahAz para kuCa logoM ko romAncakArI nOkAyAna ke lie praSikRiwa kareMge, jisase vaha apanI jIvikA kA sAXana BI banA sakegeM.
magara naxI kA vAswavika SAnwa svarUpa, usakI vAswavika sunxarawA hE.
usI kA pramANa hE ki aba BI vixeSI paryataka BArawa kI kalA ko na kevala xeKane Awe hEM, balki usake Age nawamaswaka BI ho jAwe hEM.
agara Apa vixeSa se cennaI ke lie udZAna Bara rahe hEM wo hAive para aMwararARtrIya havAI addA AwA hE.
uwwarAyaNa sUrya ke samaya 21 jUna ko yahAz sUrya nahIM CipawA hE waWA cObIsoM GaMte xina rahawA hE.
yaha Sahara Arkatika sarkala ke uwwara meM eka tApU para basA hE waWA cAroM ora pahAdZoM, PyUrdo waWA xvIpoM se GirA hE.
yaxi Apa svacCa havA, nirmala pAnI Ora KUbasUrawa vAxiyoM ke murIxa hEM wo skOYtalEMda ApakI maMjila hE.
yaxi Apa BIdZa-BAdZa se xUra ekAMwa meM prakqwi kI anupama CatA ko nihAranA cAhawe hEM wo skOYtalEMda ke pAsa Apako xene ke lie kAPI kuCa hE.
yahAz golPa KilAyA hI nahIM jAwA balki siKAyA BI jAwA hE.
pahalI maMjila para SAnaxAra hotala hE, xUsarI para  kEPe Ora wIsarI maMjila ko suMxara najAroM ko nihArane ke lie KAlI CodZA gayA hE.
mIrA yahAz apane kqRNa kI pUjA karawI WI Ora yahIM para unhoMne amqwa mAnakara viRa kA pyAlA pI liyA WA.
swUpa kramAMka xo pahAdZI ke SiKara para sWiwa hE Ora yaha pawWara kI suMxara cAraxIvArI se GirA huA hE.
yahAM lagaBaga 75 maMxira hEM Ora inake bAre meM alaga-alaga kiMvaxaMwiyAM hEM.
varRaBara parvawIya SiKara varRA se AcCAxiwa rahawe hEM jabaki mExAnI BAga  regiswAna hE.
kiMvaxaMwiyoM ke anusAra Siva SaMBu amara hEM, kiMwu sawI ke sAWa jIvana-maraNa kA baMXana calawA rahawA WA.
paryatana manwrAlaya kA mAnanA hE ki isa waraha ke romAncakArI KeloM ko baDZAvA xenA bahuwa AvaSyaka hE.
EsA lage ki hama BagavAna SrI hari viRNu ke nagara meM pahuzca gae hEM.
yahAz para hI viSva prasixXa rAma JUlA evaM lakRmaNa JUlA nAmaka pula hEM, jo gaMgA naxI para bane hEM.
gaMgA ke nirmala jala kI kala-kala Xvani se mugXa kara xene vAlA saMgIwa pExA howA hE, jo yahAz Ane vAle hara vyakwi ko apanI ora AkarRiwa karawA hE.
jisa sWAna para naMxI bEla ko CodZA gayA WA, usa sWAna kA nAma bElagAzva padZA.
isa yAwrA meM hajAroM sAXu waWA SraxXAlu BI SAmila howe hEM jo BagavAna Siva ke liMga ke xarSana karane kI icCA lie howe hEM.
maMxira kI cAraxIvArI para lakadZI ke kAma kI nakkASI kI gaI hE jo saxiyoM purAnI bawAI jAwI hE.
jaba ve maxurE kI sImA meM pahuzce wo unheM sUcanA milI ki vivAha saMpanna ho gayA.
jaba yaha praKyAwa vanaspawi vEjFAnika pEroteta kI xeKa-reKa meM AyA wo yahAz kalakawwA, maxrAsa, sIlona waWA rUmAniyA se pOXe lAe gae.
yaha CotA sA  suMxara  gAzva  hE,  jahAz  se  kaI gleSiyaroM kI yAwrA kI jA sakawI hE.
jo loga AgarA Sahara meM nirmiwa sWApawya kI bejodZa kalAoM ko xeKane Awe hEM, ve eka bAra Pawehapura sIkarI kA ruKa jarUra karawe hEM.
jo BI Bakwa ina maMxiroM meM xarSana hewu Awe hEM, ve inase pUjA jarUra karavAwe hEM.
jaba pUrA rAjasWAna garmI meM wapawA hE Ora sAre jalasrowa sUKa jAwe hEM, waba BI isa gOmuKa se nikalane vAlI XArA lagAwAra SivajI kA aBiReka karawI rahawI hE.
yaha CotA sA  suMxara  gAzva  hE,  jahAz  se  kaI gleSiyaroM kI yAwrA kI jA sakawI hE.
yahAz ke brAigana GAta para lakadZI ke aneka suMxara makAna hEM, jo maXyakAlIna yuga kI yAxa xilAwe hEM.
yaha maMxira usI sWAna para banAyA gayA hE, jahAz se yaha mUrwi nikAlI gaI WI.
maMxira ke cAra badZe-badZe xvAra hEM, jo saBI xiSAoM kI ora banAe gae hEM.
TIka sAmane mOjUxa pEgodA ke Upara bazXI GaMtiyAz suramya Ora SAMwa vAwAvaraNa meM jaba bajawI hEM wo lagawA hE ki ye saBI xiSAoM meM ahiMsA Ora prema kA saMxeSa xe rahI hoM.
xakRiNa kA xarpa-Silpa sOMxaryayukwa Bavya mInAkRI maMxira xeKane ke bAxa lagA aba waka hama jinheM Bavya kahawe Ae We, ve  isake pAsaMga BI nahIM hEM.
prawyeka gopurama nO maMjilA hE, jo ki eka badZe AyawAkAra meM sWiwa hE.
agale xina julUsa vEgA naxI wata para jAwA hE, jahAz alagAra maMxira se alagArajI kI prawimA naxI ke xUsare wata para lAkara vApasa le jAI jAwI hE.
mUrwi kI pqRTaBUmi meM woraNa hE, jisa para xasa avawAroM kI lIlA ciwriwa hE.
cambA meM BUrIsiMha nAma kA eka saMgrahAlaya hE, jahAM cambA GAtI kI hara kalA suSoBiwa hE.
isa waraha ke nOkAyAna meM kaBI-kaBI naI-naI jagahoM kA BI pawA cala jAwA hE, jo apane Apa meM bilkula alaga hE.
'harixvAra' kA arWa hE eka EsA sWAna, jahAz pahuzcawe hI eka alaga-sI anuBUwi ho.
nagara meM cAroM ora BagavAna ke Bajana gUzjawe rahawe hEM.
sAla Bara yahAz SraxXAlu Awe rahawe hEM.
rakRA baMXana ke najaxIka Awe-Awe yahAz SraxXAluoM kI BIdZa kAPI baDZa jAwI hE.
amaranAWa kI guPA meM hara sAla eka prAkqwika himaliMga banawA hE.
sOBAgya se BIwara ke maMxira baca gae We.
pAMdicerI meM bOYtanikala gArdana 1826 meM banA.
wrayoxaSI ke xina puMCa kasbe ke xaSanAmI aKAdZe se isa XarmasWala ke lie CadZI mubAraka kI yAwrA AraMBa howI hE.
himAlaya se nikalI gaMgA naxI yahAz se mExAnoM kI waraPa pravAhiwa howI hE.
sa 1996 meM prAkqwika ApaxA kI vajaha se guPA ke sAmane hajAroM logoM kI mOwa ho gaI.
jahAz wawkAlIna bOxXa BikRu nivAsa karawe We.
isake  bAxa SaMBu ne mAz ko amarawA kI kaWA sunAI.
yaha pAvana sI subaha kiwanA sukUna xewI hE.
SrI bAbA amaranAWa SrAina borda ke xvArA unheM caDZAve meM se  kuCa  rASi  xI jAwI hE.
guPA ko 1950 meM bUtA malika nAmaka muslima caravAhe ne KojA WA.
vahAz para unhoMne BagavAna SrIgaNeSa ko virAjiwa kiyA.
senA isakA saMcAlana karawI hE.
hara smAraka eka kahAnI bayAna karawA hE.
merItAima myUjiyama meM Apa awIwa ke badZe-badZe jahAja xeKa sakawe hEM.
rANA sAMgA ne yahAz rahakara apanI POja ko majabUwa kiyA.
yUnesko ne ise vESvika Xarohara GoRiwa kiyA hE.
usane Bavya imArawoM se maxurE ko sazvArA.
isakI puRti isase BI howI hE ki muslima bahula kRewra meM sWiwa isa maMxira kI raKavAlI muslima hI karawe hEM.
usI ke smAraka ke rUpa meM yahAz samrAta aSoka mahAna ne eka swUpa kA nirmANa BI kiyA.
suMxarawA, rahasya, Xarma evaM viciwra SAMwi kA yaha kRewra paryatakoM ko moha lewA hE.
harixvAra ke vAwAvaraNa meM anUTI paviwrawA Ora XArmikawA najara AwI hE.
isake bAxa se logoM kI baDZawI BIdZa Ora vAxI meM baDZawe AwaMkavAxa ke kAraNa sarakAra ne yahAz surakRA muhEyA karavAI.
ina wUPAnoM se preraNA lekara oralEMdoM vAsiyoM ne eka anUTe Bavana kA nirmANa karavAyA hE Ora use nAma xiyA  hE - vandara varksa.
yahAz kI sarakAra hIroM ke vyavasAya ko pramota karane ke lie yahAz PrI vijita karavAwI hE.
salIma siMha kI havelI ko jEsalamera ke praXAnamaMwrI salIma siMha ne banavAyA WA.
yaha iwihAsa wo bacce-bacce kI jubAna para hE ki mugala bAxaSAha SAhajahAz ne apanI xUsarI pawnI  mumawAja mahala kI yAxa meM wAjamahala kA nirmANa karavAyA WA.
isakA nirmANa 1565 I. meM akabara ne karavAyA WA.
bAbara ne isa bAga ko 1528 I. meM banavAyA WA.
isa  swaMBa kA nirmANa mahArANA kuMBA ne cOxahavIM SawAbxI meM karavAyA WA.
yaha mahala rAjA bIra siMha jUxeva ne 17 vIM SawAbxI meM jahAzgIra ke oraCA Ane ke pahale banavAyA WA.
cawurBuja ke AkAra kA yaha mahala 17 vIM SawAbxI meM maXukara SAha ne banavAyA WA.
mahArAjA ne hI yahAz paxama wAlAba, rAjA bAzXa Ora milaka wAlAba banavAe.
bAxa meM bAhara vAle badZe gopura Pira banavAe gae.
yaha maMxira yahAz ke cola rAjAoM ne banavAyA WA.
133 Puta UMcI isa prawimA ko lagaBaga pAzca hajAra Silpa karmiyoM ne behaxa mehanawa se banavAyA WA.
isake sAWa hI sena naxI ke kinAroM para cahalakaxamI karawe hue PreMca jIvana se rUbarU honA behaxa suMxara anuBava hE.
badZe Ora baccoM xonoM  ko XyAna raKakara vOYtara slAidsa banAI gaI hEM.
mInAkRI maMxira ko keMxra meM raKakara nagara kI racanA kamala ke PUla kI Akqwi meM kI gaI hE.
sAWa hI isa waraha ke romAncakArI KeloM kI viSeRa vyavasWA xvArA paryatana ke sAWa-sAWa sWAnIya logoM ke liye rojagAra kI BI aneka  samBAvanAez uBara kara AegIM.
eka sWAna para Akara BagavAna BolenAWa ne naMxI bEla se uwarakara use vahIM CodZa xiyA Ora mAwA ke sAWa Age kI ora nikala padZe.
yaha leKA-joKA hajAroM varRoM se varwamAna paMdiwoM ke pUrvajoM ke xvArA sazBAlakara raKA gayA hE.
yahAz se pahAdZoM ke bIca se bahawI huI gaMgA naxI kA xarSana badZA manorama prawIwa howA hE.
nOkA meM bETakara yA kinAre para KadZe hokara maCaliyoM kA xAnA KilAnA BI avismaraNIya anuBava rahegA.
mana SAMwa, wAjA Ora praPulliwa ho jAwA hE.
ise hI bAxa meM pahalagAma kahA jAne lagA.
pahAdZoM se GirA hone ke kAraNa yaha nagara awyaMwa hI suMxara lagawA hE.
harixvAra ko BArawa kI XArmika rAjaXAnI mAnA jAwA hE.
isa sWAna kA nAma cAzxavAdZI padZA.
isa mUrwi ke cAra muKa va cAra hAWa hEM.
kahA jAwA hE ki rAnI ke prawixina narmaxA xarSana ke paScAwa anna-jala grahaNa karane kI Axawa ke kAraNa bAja bahAxura ne yaha UzcA mahala banavAyA WA .
piCale panxraha varRoM se isa kRewra meM vAhana calA rahe BAnuprawApasiMha ne bawAyA ki paryatakoM kI saMKyA niraMwara baDZa rahI hE , lekina bAGa kaBI kaBAra hI xiKawA hE .
unhoMne bawAyA ki pahale yahAz bAGa xiKAne ke lie cAra mArga We , aba sAwa kara xie gae hEM .
Kule vAhanoM meM binA kisI haWiyAra ke paryatakoM ko jaMgala meM le jAne ke bAvajUxa xurGatanA kI ASaMkA se inakAra karawe hue eka anya vAhana cAlaka ne bawAyA ki yaxi jaMgala meM gAdZI meM koI KarAbI ho jAe wo isakI sUcanA xene ke lie cAlaka ko pExala hI cOkI waka jAnA howA hE aWavA rAswe se gujara rahA koI anya vAhana isakI Age sUcanA xewA hE .
kahA jAwA hE ki ina swUpoM meM buxXa Ora unake SiRyoM ke avaSeRa raKe gae hEM .
wiruvanaMwapurama kuCa vEjFAnika Ora wakanIkI saMsWAnoM kA BI kenxra hE jinameM vikrama sArABAI spesa seMtara , seMtara POYra arWa sAiMsa stadIja Ora eka EsA saMgrahAlaya hE joki vijFAna Ora prOxyogikI ke saBI pahaluoM se sAkRAwkAra karAwA hE .
himAlaya meM paScima gaDZavAla ke barPa se Dazke SraMga baMxarapucCa jo ki jamIna se 20 , 731 Puta UzcA hE , ke uwwara-paScima meM kAliMxa parvawa hE .
viSeRaNa AkarRaNa yaha hE ki isa maMxira ke pAsa paviwra gaMgA naxI jo ki baMgAla meM hugalI naxI ke nAma se jAnI jAwI hE , bahawI hE .
prawyeka gopurama nO maMjilA hE , jo ki eka badZe AyawAkAra meM sWiwa hE .
